ui/DesignSystem/Component/BigHeader.svelte

```diff
@@ -0,0 +1,97 @@
+<script>
+  export let style = null;
+  export let data = null;
```

Comment:
This should probably be `entity` as it's either an `org` or `user`.

Reply:
Addressed in [81a3378](https://github.com/radicle-dev/radicle-upstream/pull/494/commits/81a33787568db5f5a6247813b280e58a7a590afa)